test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala

```diff
@@ -93,6 +94,36 @@ class CheckLiabilityDetailsAnswersControllerTest extends ControllerSpec {
         )
       }
 
+      "group registration enabled and group of organisation is selected" in {
+        authorizedUser(features = Map(Features.isGroupRegistrationEnabled -> true))
+
+        val registration = aRegistration(withRegistrationType(Some(RegType.GROUP)))
+        mockRegistrationFind(registration)
+        given(page.apply(refEq(registration), any(), any())(any(), any())).willReturn(
+          HtmlFormat.empty
+        )
+        when(mockStartRegistrationController.startLink(any())).thenReturn(startLiabilityLink)
+
+        verifyExpectedLinks(backLink = routes.MembersUnderGroupControlController.displayPage().url,
+                            changeLiabilityLink = startLiabilityLink.url
+        )
+      }
+
+      "group registration enabled with no reg type selected" in {
+        authorizedUser(features = Map(Features.isGroupRegistrationEnabled -> true))
+
+        val registration = aRegistration(withRegistrationType(None))
+        mockRegistrationFind(registration)
+        given(page.apply(refEq(registration), any(), any())(any(), any())).willReturn(
+          HtmlFormat.empty
+        )
+        when(mockStartRegistrationController.startLink(any())).thenReturn(startLiabilityLink)
+
+        verifyExpectedLinks(backLink = routes.RegistrationTypeController.displayPage().url,
+                            changeLiabilityLink = startLiabilityLink.url
+        )
+      }
+
```

Comment:
Wondering whether we ought to combine these 3 tests into 2: both for when group registration enabled but one checking single entity behaviour and the other for groups?

Reply:
I think the first one is already testing for Single entity and the second one is testing for Group. The third one was testing for None but seeing the case statement looks like 2 are enough. Dont need the third, have deleted it